routers/api/v1/org/team.go

```diff
@@ -42,6 +42,35 @@ func ListTeams(ctx *context.APIContext) {
 	ctx.JSON(200, apiTeams)
 }
 
+// ListUserTeams list all the teams a user belongs to
+func ListUserTeams(ctx *context.APIContext) {
+	// swagger:operation GET /user/teams user userListTeams
+	// ---
+	// summary: List all the teams a user belongs to
+	// produces:
+	// - application/json
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/TeamList"
+	teams, err := models.GetUserTeams(ctx.User.ID)
+	if err != nil {
+		ctx.Error(500, "GetUserTeams", err)
+		return
+	}
+
+	apiTeams := make([]*api.Team, len(teams))
+	for i := range teams {
+		org, err := models.GetUserByID(teams[i].OrgID)
```

Comment:
For better performance, maybe you could add a user cache via `var cache = make(map[int64]*User)`, if user are in many teams of one org that will reduce database queries.

Reply:
Agree! :)